documentation/assemblies/deploying/assembly-deploy-kafka-connect.adoc

```diff
@@ -8,23 +8,19 @@
 
 [role="_abstract"]
 link:https://kafka.apache.org/documentation/#connect[Kafka Connect^] is a tool for streaming data between Apache Kafka and external systems.
+Kafka Connect is typically used to integrate Kafka with external databases and storage and messaging systems.
```

Comment:
I'm not sure I would put it this way. It is one of the use cases, sure. But it seems like a strong statement that might lead people away from using Connect to integrate with other systems than those mentioned here.

Reply:
Right. I've made these examples: _...Apache Kafka and other systems.
For example, Kafka Connect might integrate Kafka with external databases or storage and messaging systems._